etc/kayobe/environments/ci-multinode/hooks/overcloud-host-configure/pre.d/20-reboot.yml

```diff
@@ -0,0 +1 @@
+../../../../../ansible/reboot.yml
```

Comment:
This is a bad idea. It would hook a reboot into every host configure command, and would be pretty surprising and potentially damaging. Is there another way this can be fixed?

Reply:
Perhaps apply the reboot from the fix-hostname.yml playbook if a change in hostname is detected?